accord-core/src/test/java/accord/impl/basic/TaskExecutorService.java

```diff
@@ -44,14 +44,19 @@ public Task(Callable<T> fn)
         @Override
         public void run()
         {
+            T result = null;
```

Comment:
to avoid copy/paste maybe we should `extends AsyncResults.RunnableResult`?  

```
public static class Task<T> extends AsyncResults.RunnableResult<T> implements Pending, RunnableFuture<T>
    {
        public Task(Callable<T> fn)
        {
            super(fn);
        }
```

we can avoid the run method in this case completely